handbook/engineering/releases/release_issue_template.md

```diff
@@ -31,12 +30,17 @@ Arguments:
   yarn run release changelog:cut $MAJOR.$MINOR.0
   ```
 - [ ] Create the `$MAJOR.$MINOR` branch off the CHANGELOG commit in the previous step: `git branch $MAJOR.$MINOR && git push origin $MAJOR.$MINOR`.
+
+## Release candidates
+
+Upon branch cut, cut and test the first release candidate:
```

Comment:
```suggestion
Upon branch cut, create and test the first release candidate:
```

Reply:
Added this suggestion, but we seem to use a mix of "cut" and "create" for creating releases - probably not very important to unify the terminology though I suppose